### ISSUE TITLE
Class based component example and re-render

### ISSUE THREAD

**Issue by synapse** [arnelenero/simpler-state#13]
Hey there, very nice library. Super easy to set up and ready to be used.
My question is since I still have a mixed function and calss based components, if is there a way to trigger a re-render when the store is update?

**Comment by arnelenero** [comment]
Hi.  Since simpler-state's API was primarily designed to be used with Hooks, the class-based support has remained undocumented.  But if you're not using TypeScript, there is one method on the entity that you can use, something like this: inside your class-based component:

In an initializer function:

```javascript
this.unsubscribeToCounter = counter._subscribe(value => {
  if (this.state.counter !== value) 
    this.setState({ counter: value })
})
```

In a cleanup function:
```javascript
   this.unsubscribeToCounter()
```

**Comment by synapse** [comment]
Thanks, @arnelenero that would work. I'll have to make an interface since I'm using TS. But in the meanwhile, I found out that I could also create a function-based component and wrap my main component passing down the store as props.
Anyways keep up the good works. Seams nice to use

**Comment by arnelenero** [comment]
> Thanks, @arnelenero that would work. I'll have to make an interface since I'm using TS. But in the meanwhile, I found out that I could also create a function-based component and wrap my main component passing down the store as props.

Good point, that would be the quickest workaround indeed, especially if you're using TS.
